src/features/duplicates/components/PotentialDuplicatesLists.tsx

```diff
@@ -0,0 +1,91 @@
+import { Box, Typography } from '@mui/material';
+import { DataGridPro, GridColDef } from '@mui/x-data-grid-pro';
+import { FC, useState } from 'react';
+
+import getColumns from '../utils/getColumns';
+import messageIds from '../l10n/messageIds';
+import { PotentialDuplicate } from '../store';
+import useDuplicatesMutations from '../hooks/useDuplicatesMutations';
+import { useMessages } from 'core/i18n';
+import { useNumericRouteParams } from 'core/hooks';
+import { ZetkinPerson } from 'utils/types/zetkin';
+
+interface PotentialDuplicatesListsProps {
+  duplicate: PotentialDuplicate;
+}
+
+const PotentialDuplicatesLists: FC<PotentialDuplicatesListsProps> = ({
+  duplicate,
+}) => {
+  const messages = useMessages(messageIds);
+  const { orgId } = useNumericRouteParams();
+  const { addDuplicatePerson, removeDuplicatePerson } =
+    useDuplicatesMutations(orgId);
+  const rowsToMerge: ZetkinPerson[] = duplicate?.duplicates || [];
+  const [rows, setRows] = useState<ZetkinPerson[]>([]);
```

Comment:
This state could maybe be named "peopleNoToMerge", so it's a bit more clear what it contains?